questions.csv

```diff
@@ -5,7 +5,7 @@ conflict_cooperation,"At the end of this task, one or more group members will be
 conflict_interests,"During this task, different group members have conflicting interests, objectives, or goals. ",task,Strongly disagree | Somewhat disagree | Neither agree nor disagree | Somewhat agree | Strongly agree,ordinal,McGrath 1984
 effort_mental,This task requires only mental effort and no physical effort.,task,Strongly disagree | Somewhat disagree | Neither agree nor disagree | Somewhat agree | Strongly agree,ordinal,Shaw 1963
 effort_physical,This task requires only physical effort and no mental effort.,task,Strongly disagree | Somewhat disagree | Neither agree nor disagree | Somewhat agree | Strongly agree,ordinal,Shaw 1963
-external_adversary,The group doing the task has a common opponent or adverary (individual or group) that is external to the group.,task,Strongly disagree | Somewhat disagree | Neither agree nor disagree | Somewhat agree | Strongly agree,ordinal,McGrath 1984
+external_adversary,The group doing the task has a common opponent or adversary (individual or group) that is external to the group.,task,Strongly disagree | Somewhat disagree | Neither agree nor disagree | Somewhat agree | Strongly agree,ordinal,McGrath 1984
```

Comment:
I misunderstood and thought you meant something in the issue template. This is actually fixed from a google doc that sources all our questions, so I made the update there, and then pulled everything in again, which can sort of be done by running the r script in `/analysis`. 

I say sort of because we have a glitch service that reads the google doc on boot (and is where the task mapping survey is hosted), but we can't trigger it to reboot externally.  So if its not running we can then call it from R and it will boot and pull in the clean version, otherwise, we need to manually reboot it which takes messing with glitch.

Anyway all fixed now.